pytket/extensions/pennylane/pytket_device.py

```diff
@@ -36,14 +36,14 @@ class PytketDevice(QubitDevice):
 
     name = "pytket-pennylane plugin"
     short_name = "pytket.pytketdevice"
-    pennylane_requires = ">=0.14.0"
-    version = "0.14.0"
+    pennylane_requires = ">=0.35.0"
```

Comment:
Is this something we track in the changelog?

Reply:
(We do track dependency updates separately. (I'm not sure this has any significance.)